tools/model-compiler/src/main/java/io/spine/tools/gradle/compiler/ModelCompilerPlugin.java

```diff
@@ -63,8 +63,8 @@ public void apply(Project project) {
     }
 
     private void apply(SpinePlugin plugin, Project project) {
-        _debug("Applying {}", plugin.getClass()
-                                         .getName());
+        _debug().log("Applying `%s`.", plugin.getClass()
```

Comment:
Maybe we can rely on the default `class#toString` instead?

Reply:
`Class.toString()` prepends “class” or ”interface”. We don't want it inside the backticked fragment.